frontend/src/components/DefenceBox/DefenceConfigurationInput.tsx

```diff
@@ -20,7 +20,7 @@ function DefenceConfigurationInput({
 }) {
 	const CONFIG_VALUE_CHARACTER_LIMIT = 5000;
 	const [value, setValue] = useState<string>(currentValue);
-	const [configInvalid, setConfigInvalid] = useState<boolean>(false);
+	const [inputInvalid, setInputInvalid] = useState<boolean>(false);
```

Comment:
This was fine as "configInvalid" because this component is all about Configuration. It was just the Themed components where I was suggesting changing the prop name to "input" or "value".

Reply:
changed the state back to a positive declaration and passed the negation thereof to the inputs